src/EnergyPlus/AirLoopHVACDOAS.hh

```diff
@@ -164,15 +165,9 @@ namespace AirLoopHVACDOAS {
         std::vector<std::string> AirLoopName;
         std::vector<int> m_OACtrlNum; // array of OA controller number
 
-        int HWLoopNum;
-        DataPlant::LoopSideLocation HWLoopSide;
-        int HWBranchNum;
-        int HWCompNum;
```

Comment:
Individual plant location variables: `LoopNum`, `LoopSide`, `BranchNum`, and `CompNum` were combined into a Plant Location variable. The individual variables can still be accessed using the `.`operator, for eg: `HWPlantLoc.loopNum`.